regen-network/regen-web
# feat: APP-39 draft project tag

## Description

https://regennetwork.atlassian.net/browse/APP-39?atlOrigin=eyJpIjoiNTE0MmY4OWI4Yjg3NGY5ZTg2ZDg4Njg4YzNhYTEzMjQiLCJwIjoiaiJ9



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2341--regen-marketplace.netlify.app/profile/projects login and check the unpublished projects are marked with the tag "draft"

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
